src/e2e-test/java/io/cdap/plugin/bigquery/stepsdesign/BigQueryBase.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright © 2021 Cask Data, Inc.
+ * Copyright © 2023 Cask Data, Inc.
```

Comment:
why this change? Looks like nothing is changed in this file.

Reply:
An unintended step was initially removed and is now added. Aryan has also suggested changing the year to 2023 as part of the same change.

Reply:
you said some step definition was added, but I still don't anything changed in this file, am I missing something?

Reply:
I Apologize for the misunderstanding. The changes were made to the GCS Base, not the BigQueryBase. Reverted the year back to 2021. [commit link](https://github.com/data-integrations/google-cloud/pull/1273/commits/7730c6e09d09620852af5908b7827d9d56a416e3)